csrc/scheduler/pointwise.cpp

```diff
@@ -295,7 +295,10 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
       });
 
   int64_t max_dtype_size_bit_for_vectorization = 0;
+  // ugly WAR.
```

Comment:
Please give more details rather than just saying it's ugly. Why is it ugly? What should have been done instead?

Also, this variable is only used hundreds of lines below. For better code readability, it should be moved down there.